### PR TITLE
Parse tsc error output directly to the console

### PR DESCRIPTION
**Pull request by tlbdk**

### CODE REVIEW

**Comment by tlbdk** [comment]
https://app.clubhouse.io/connectedcars/story/32235/make-error-output-from-node-setup-build-prettier

File: src/build.ts [hunk 1]
```diff
@@ -99,6 +99,28 @@ export async function babelBuild(rootDirs: string[], outDir: string): Promise<Bu
   return buildList
 }
 
+interface ExeFileError extends Error {
```
Comment (chribsen) [inline src/build.ts:102]:
```suggestion
interface ExecFileError extends Error {
```

File: bin/build.ts [hunk 2]
```diff
@@ -40,5 +40,10 @@ main(process.argv)
     process.exit(exitCode)
   })
   .catch(e => {
-    console.error(e)
+    if (e instanceof BuildErrorOutput) {
+      console.error(e.message)
+    } else {
+      console.error(e)
```
Comment (chribsen) [inline bin/build.ts:46]: Can't you always do `console.error(e.message)`? It's a property on `Error`

Comment (chribsen) [reply bin/build.ts:46]: I got it now.

**Review by chribsen — APPROVED** [review]

File: bin/build.ts (hunk 2 above)
Comment (tlbdk) [reply bin/build.ts:46]: If it is unexpected I want the stack trace and extra information

**Pull request merged**